### PR TITLE
fix: arkeo claim and arkeo ts-client

### PR DESCRIPTION
**Pull request by shreyasbhat0**

### CODE REVIEW

File: src/hooks/useClaim.tsx [hunk 1]
```diff
@@ -47,18 +47,16 @@ export const useClaim = () => {
         }
         result = await client.ArkeoClaim.tx.sendMsgClaimEth({
           value: {
+            creator: arkeoAccount,
```
Comment (toshiSat) [inline src/hooks/useClaim.tsx:50]: We need to keep thorTx, it allows for claiming thorchain transactions

File: ts-client/arkeo.arkeo/index.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
-import Module from './module';
+import IgntModule from './module';
```
Comment (to-sh-1) [inline ts-client/arkeo.arkeo/index.ts:1]: ts-client is generated right?  Is this just a fork of ignite's repo?

**Review by to-sh-1 — COMMENTED** [review]
overall lgtm, thank you for making the string changes for creator

File: src/hooks/useClaim.tsx [hunk 3]
```diff
@@ -47,18 +47,16 @@ export const useClaim = () => {
...
             ethAddress: ethAccount,
-            signature,
-            thorTx: thorDelegateTx ?? '',
-            creator,
```
Comment (toshiSat) [inline src/hooks/useClaim.tsx:53]: We need to keep thorTx, it allows for claiming thorchain transactions

**Review by to-sh-1 — APPROVED** [review]

File: ts-client/arkeo.arkeo/index.ts (hunk 2 above)
Comment (shreyasbhat0) [reply ts-client/arkeo.arkeo/index.ts:1]: Generated but as the ignite issues still persists this has been changed manually

**Pull request merged**